ios/RNZoomUs.m

```diff
@@ -239,4 +243,24 @@ - (void)onMeetingError:(MobileRTCMeetError)errorCode message:(NSString *)message
   meetingPromiseReject = nil;
 }
 
+- (void)onClickShareScreen:(UIViewController *)parentVC {
+  if (@available(iOS 12.0, *)) {
+    CGRect frame = parentVC.view.bounds;
+    RPSystemBroadcastPickerView *pickerView = [[RPSystemBroadcastPickerView alloc] initWithFrame:frame];
+    pickerView.preferredExtension = screenShareExtension;
+    SEL buttonPressed = NSSelectorFromString(@"buttonPressed:");
+    if ([pickerView respondsToSelector:buttonPressed]) {
+      [pickerView performSelector:buttonPressed withObject:nil];
+    }
+  }
+}
+
+- (BOOL)respondsToSelector:(SEL)aSelector
```

Comment:
Might be more clear if we call this `resolveIfScreenShareExtension`

Reply:
We can't rename this because we're overriding an inherited method.